packages/synapse-react-client/src/components/AccountLevelBadges/AccountLevelBadges.test.tsx

```diff
@@ -1,28 +1,49 @@
 import { UserBundle } from '@sage-bionetworks/synapse-types'
-import { render, screen, waitFor } from '@testing-library/react'
+import { render, screen, waitFor, within } from '@testing-library/react'
 import React from 'react'
-import { mockUserProfileData } from '../../mocks/user/mock_user_profile'
+import {
+  mockUserBundle,
+  mockUserProfileData,
+} from '../../mocks/user/mock_user_profile'
 import SynapseClient from '../../synapse-client'
 import { createWrapper } from '../../testutils/TestingLibraryUtils'
 import {
-  AccountLevelBadgeType,
   accountLevelBadgeConfig,
+  AccountLevelBadgeType,
 } from '../AccountLevelBadge/AccountLevelBadge'
 import { AccountLevelBadges } from './AccountLevelBadges'
+import { MOCK_ACCESS_TOKEN } from '../../mocks/MockSynapseContext'
+import userEvent from '@testing-library/user-event'
+
+const USER_ID = '345424'
 
 jest.mock('../../synapse-client', () => ({
+  getMyUserBundle: jest.fn(),
   getUserBundle: jest.fn(),
+  revokeCertification: jest.fn(),
 }))
 
+const mockGetMyUserBundle = jest.mocked(SynapseClient.getMyUserBundle)
```

Comment:
Need to now mock the callers' user bundle, because we need to see if they are ACT